src/Redux/actions/ArmySquadChoice.test.js

```diff
@@ -0,0 +1,18 @@
+import store from '../store';
+import handleInitialArmy from '../../utilities/handleInitialArmy';
+import ArmySquadRowAdd from './ArmySquadRowAdd';
+import RolesChange from './RolesChange';
+import ArmySquadChoice from './ArmySquadChoice';
+
+const mockRowAdd = { roleIndex: 0, rowIndex: 0 };
+const mockPayload = { roleIndex: 0, rowIndex: 0, SquadID: 'PirateSquad' };
+const mockRole = [{ id: '99999', Name: 'BestRole' }];
+
+test('Redux Action ArmySquadChoice is successful in redux state change', () => {
+  store.dispatch(RolesChange(mockRole));
+  handleInitialArmy();
+  store.dispatch(ArmySquadRowAdd(mockRowAdd));
+  store.dispatch(ArmySquadChoice(mockPayload));
+  const ArmyState = store.getState().army.SquadArray;
+  expect(ArmyState[0].Squads[0].Squad).toBe('PirateSquad');
```

Comment:
For all my tests on the website you would like a space before the expect?  I have 190 tests

Reply:
Well moving forward you should try to space your tests out based on the three As of testing for readabilty (Arrange, Act, Assert). For example.

```javascript
// Variables up here

// Interact with varaibles

// Assertion goes here
```

Spaces are super important for readability between code chunks. Line breaks give people unfamiliar with something a quick way to process it in chunks. For example, a book without paragraphs would be frustrating to read. I guess the point here is you should move forward with trying to use line breaks to separate out your code visually. I really struggled to quickly read some of the tests because of it.